sdk/table/azure-table/tests/test_table.py

```diff
@@ -116,6 +116,32 @@ def test_create_table_fail_on_exist(self, resource_group, location, storage_acco
         # self.assertEqual(existing[0], [table_name])
         ts.delete_table(table_name)
 
+    @GlobalStorageAccountPreparer()
+    def test_create_table_invalid_name(self, resource_group, location, storage_account, storage_account_key):
+        # Arrange
+        ts = TableServiceClient(self.account_url(storage_account, "table"), storage_account_key)
+        # table_name = self._get_table_reference()
+        # btable_client = ts.get_table_client(table_name)
+        invalid_table_name = "$&#*_(%&@*(_("
```

Comment:
i wonder if it'd be good to switch the invalid table name to the one that you initially found this error with. i.e. 'my-table', since it's a more common mistake for users to make. Sorry should've mentioned this one earlier

Reply:
I think you're right, I'll change that. Users aren't as likely to make a table name `$&#*_(%&@*(_(` as they are `my_table`